examples/Example/src/swipeable/GmailStyleSwipeableRow.tsx

```diff
@@ -9,7 +9,7 @@ const AnimatedView = Animated.createAnimatedComponent(View);
 
 export default class GmailStyleSwipeableRow extends Component {
   renderLeftActions = (
-    _progress: Animated.AnimatedInterpolation,
+    progress: Animated.AnimatedInterpolation,
```

Comment:
Here the underscore meant that the value is not used. I think we don't want to change it? Same below and in some other places.

Reply:
Yeah I guess i only meant instance fields that were marked sort of private in plain js style

Reply:
Yes, it is rather a mistake made by @jakub-gonet and should be reverted.

Reply:
Seems like regex did too much good 😅

Eslint didn't report any of those, we should check it's config.